src/app/series/directives/series-plan.component.spec.ts

```diff
@@ -0,0 +1,134 @@
+import { cit, create, provide, cms } from '../../../testing';
+import { SeriesModel } from '../../shared';
+import { SeriesPlanComponent } from './series-plan.component';
+import { TabService, SimpleDate } from 'ngx-prx-styleguide';
+
+fdescribe('SeriesPlanComponent', () => {
+
+  create(SeriesPlanComponent);
+
+  provide(TabService);
+
+  let series, tpl1, tpl2, dist1, dist2;
+  beforeEach(() => {
+    series = cms.mock('prx:series', {title: 'mock-series'});
+    tpl1 = cms.mock('prx:tpl1', {label: 'one', _links: {self: {href: '/link/one'}}});
+    tpl2 = cms.mock('prx:tpl2', {label: 'two', _links: {self: {href: '/link/two'}}});
+    dist1 = cms.mock('prx:dist', {kind: 'somethingelse'});
+    dist2 = cms.mock('prx:dist', {kind: 'podcast'});
+    series.mockItems('prx:distributions', [dist1, dist2]);
+    series.mockItems('prx:audio-version-templates', [tpl1, tpl2]);
+  });
+
+  // cit('loads the series templates', (_fix, _el, comp) => {
```

Comment:
These tests were causing karma to freeze, for some reason.  I'd like to get them back in here, but may wait until things are jest-ified.